appengine/genetics/js/cage.js

```diff
@@ -146,40 +175,38 @@ Genetics.Cage.GAME_TIME_LIMIT_MSEC = 5 * 60 * 1000;
 Genetics.Cage.FUNCTION_TIMEOUT_STEPS = 100000;
 
 /**
- * The next available player ID.
- * @type {number}
- * @private
- */
-Genetics.Cage.nextAvailablePlayerId_ = 0;
-
-/**
  * The next available mouse ID.
  * @type {number}
  * @private
  */
 Genetics.Cage.nextAvailableMouseId_ = 0;
 
 /**
- * Callback function for end of game.
- * @type {Function}
- * @private
+ * Whether fights are discrete (a bigger mouse will always win and same size
```

Comment:
Whether outcomes of fights vary deterministically (a bigger mouse will always win; same size ties) or probabilistically, based on size.

And maybe deterministicFights, not discreteFights. "Discrete" means something different.